docs/quality-standards/scorecards.yaml

```diff
@@ -701,7 +701,7 @@ Telescope:
       deployment-integration-test:
         compliant: No
       deployment-auto-pull-requests:
-        compliant: Partially
+        compliant: No
```

Comment:
driveby: we don't deploy telescope on pull-requests

I assume that if a rule has no popularity at all (like this one, we could choose to drop it)